grommet/grommet
# Drop button to also accept HTML button props

#### What does this PR do?
This PR starts the clean up of the over use of type definitions 
This PR adds to DropButton to also accept the props of HTML Button for TS
#### Where should the reviewer start?
DropButton/index.d.ts
#### What testing has been done on this PR?
manually tested by branching out to a typescript project
#### How should this be manually tested?
It should work with the following markup

```
         <DropButton
          icon={<Add/>}
          dropContent={<Text>Brittany</Text>}
          href="/test"
          label='Choose'
          open={true}
          tabIndex={-1}
          fill={false}
        />
```
#### Any background context you want to provide?

#### What are the relevant issues?
closes #3263  
#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
no
#### Should this PR be mentioned in the release notes?

#### Is this change backwards compatible or is it a breaking change?
backwards compatible
